arraycontext/container/traversal.py

```diff
@@ -779,7 +779,10 @@ def _unflatten(template_subary: ArrayOrContainerT) -> ArrayOrContainerT:
             # {{{ check strides
 
             if strict and hasattr(template_subary, "strides"):
-                if template_subary.strides != subary.strides:
+                if (
+                    (template_subary.strides != subary.strides)
+                    and (template_subary.size != 0)
```

Comment:
Looks like this patch misses the resolution from https://github.com/kaushikcfd/arraycontext/pull/4#discussion_r896022401. (Also, please explain *why* the strides checking ill-defined.)

Reply:
Yup it got dropped when I was updating my branch. Its resolved now. Thanks !